src/Request/Version.elm

```diff
@@ -23,38 +28,46 @@ type alias VersionData =
 
 
 type Version
-    = NewerVersion
+    = NewerVersion VersionData VersionData
```

Comment:
Maybe naming the types with old/new may help the readability here?

Reply:
https://github.com/MTES-MCT/ecobalyse/pull/989/commits/5cfa3a0ecffeb0799557d4ce44c8fdb4af175041